Cython/Utility/Builtins.c

```diff
@@ -303,6 +303,18 @@ static PyObject *__Pyx_PyLong_AbsNeg(PyObject *n) {
 #define __Pyx_PyNumber_Power2(a, b) PyNumber_Power(a, b, Py_None)
 
 
+//////////////////// divmod_int.proto //////////////////
+
+static CYTHON_INLINE PyObject* __Pyx_divmod_int(int a, int b) {
+    PyObject* result_tuple;
+    div_t res = div(a, b);
+    result_tuple = Py_BuildValue("(ii)", res.quot, res.rem);
```

Comment:
I'd suggest probably doing `PyLong_FromLong` on res.quot and res.rem rather than `Py_BuildValue` - parsing the string isn't likely to be hugely quick and we are trying to make this an optimization.

Reply:
Switched to it in the third commit, please review. Thank you.